api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -142,17 +141,35 @@ type NodePoolSpec struct {
 	// * ImageDigestMirrorSet
 	//
 	// This is validated in the backend and signaled back via validMachineConfig condition.
+	// Changing this field will trigger a NodePool rollout.
 	// +kubebuilder:validation:Optional
 	Config []corev1.LocalObjectReference `json:"config,omitempty"`
 
+	// tuningConfig is a list of references to ConfigMaps containing serialized
+	// Tuned or PerformanceProfile resources to define the tuning configuration to be applied to
+	// nodes in the NodePool. The Tuned API is defined here:
+	//
+	// https://github.com/openshift/cluster-node-tuning-operator/blob/2c76314fb3cc8f12aef4a0dcd67ddc3677d5b54f/pkg/apis/tuned/v1/tuned_types.go
+	//
+	// The PerformanceProfile API is defined here:
+	// https://github.com/openshift/cluster-node-tuning-operator/tree/b41042d42d4ba5bb2e99960248cf1d6ae4935018/pkg/apis/performanceprofile/v2
+	//
+	// Each ConfigMap must have a single key named "tuning" whose value is the
+	// JSON or YAML of a serialized Tuned or PerformanceProfile.
+	// Changing this field will trigger a NodePool rollout.
+	// +kubebuilder:validation:Optional
+	TuningConfig []corev1.LocalObjectReference `json:"tuningConfig,omitempty"`
```

Comment:
why was this moved within the struct def?

Reply:
sorry, you caught me again moving stuff around and making review confusing! I just thought it was better cohesively located next the .config. I'll put it back to avoid confusion.